src/lv_gpu/lv_gpu_stm32_dma2d.c

```diff
@@ -241,7 +241,8 @@ static void invalidate_cache(void)
     if(disp->driver.clean_dcache_cb) disp->driver.clean_dcache_cb(&disp->driver);
     else {
 #if __CORTEX_M >= 0x07
-        SCB_CleanInvalidateDCache();
+    	if((SCB->CCR) & (1<<((uint32_t)SCB_CCR_DC_Msk)))
```

Comment:
Correct me if I'm wrong, but don't you do the shift for the `_Pos` macros, not the `_Msk` ones (which are already a bitmask)?

Reply:
You're correct, my mistake. I've pushed the correct version into this PR.